src/dsp_tools/utils/excel_to_json/properties.py

```diff
@@ -0,0 +1,457 @@
+import importlib.resources
+import json
+import warnings
+from typing import Any, Optional
+import jsonpath_ng.ext
+import jsonschema
+import regex
+import pandas as pd
+import numpy as np
```

Comment:
Here, isort could add an empty line before pandas, and then sort the imports in each group alphabetically. (Okay, not the most important thing to be done, but I just mention it 😄 )